doc/release-notes/release-6.sgml.in

```diff
@@ -307,7 +307,7 @@ This section gives an account of those changes in three categories:
 <sect1>Changes to existing options<label id="modifiedoptions">
 <p>
 <descrip>
-	<p>There are no ./configure options changed.
+	<p>There have been no ./configure options changed.
```

Comment:
The official and the current PR versions sound garbled to me. If you want to use present perfect, please consider using these phrasing/grammar improvements/fixes instead:

```suggestion
	<p>There have been no ./configure option changes.
```

or

```suggestion
	<p>No ./configure options have been changed.
```

... or something like that.

Reply:
Sigh. I wonder if I will ever get the hang of how USA'ian grammar works.